### DISCUSSION TITLE
doris verson 2.0.2 , create a new table with a auto_increment column, but always show errors.

### DISCUSSION THREAD

**Discussion by uicats** [apache/doris#27871] [A - General / Q&A]
I need a help to resolve this problem. 

Doris create a new table cannot with auto_increment column.

My sql is like this:

CREATE TABLE IF NOT EXISTS flows (
  flow_id bigint NOT NULL AUTO_INCREMENT,
  flow_time datetime NOT NULL
  ) ENGINE = OLAP DUPLICATE KEY(
  `integral_flow_id`
) DISTRIBUTED BY HASH(`flow_time`) BUCKETS 8 PROPERTIES ("replication_num" = "3");

But Doris always show error like this:
(conn=134) errCode = 2, detailMessage = auto increment column is not supported currently.

How can I do to resolve this problem ?

**Comment by uicats** [comment]
I copy sql with a wrong, replace it. 

CREATE TABLE IF NOT EXISTS flows (
flow_id bigint NOT NULL AUTO_INCREMENT,
flow_time datetime NOT NULL
) ENGINE = OLAP DUPLICATE KEY(
`flow_id`
) DISTRIBUTED BY HASH(`flow_time`) BUCKETS 8 PROPERTIES ("replication_num" = "3");

**Reply by zhbdesign** [reply]
https://github.com/apache/doris/issues/22544

**Comment by uicats** [comment]
What time will Doris release the version 2.1 ?

**Reply by Hanchers** [reply]
是呀，2.1啥时候发布呀

**Comment by sdhzwc** [comment]
2.1啥时候发版

**Comment by yiguolei** [comment]
@sdhzwc 2.1 大概在1月15-1月20日之间发版
